README.md

```diff
@@ -76,6 +77,28 @@ var app = new EmberApp(defaults, {
 
 The function receives the `filePath` for each asset and must return a string.
 
+### Using With `broccoli-asset-rev`
```

Comment:
I'm unsure we need an example this long. I think we simply need to mention that `broccoli-asset-rev` needs to run after your addon and that you can specify that via the `ember-addon.after` field in the `package.json`.

I want to make sure the readme doesn't become overly verbose (makes it harder to locate things quickly). If you feel this much detail is definitely needed, maybe we could start up a wiki or additional docs folder.